pip/pep425tags.py

```diff
@@ -158,6 +158,8 @@ def get_supported(versions=None, noarch=False):
 
     abis.append('none')
 
+    arches = []
+
     if not noarch:
```

Comment:
Renaming this variable `arch` and flipping the values in the surrounding code would make this a bit less confusing. Double negatives FTL where avoidable